### ISSUE TITLE
✨ Docs - Last Edited to be Automatic

### ISSUE THREAD

**Issue by joshbermanssw** [tinacms/tina.io#2005]
### 🩸 Pain

A lot of our docs seem old as they have their last updated set to be very old. This could be because they haven't been updated in a while, or because people haven't manually updated it.

### 💡 Suggested Solution

Make the last updated by automatic 

### 💼 Tasks 

- [x] Investigate the beforeSubmit hook
- [x] Apply the beforeSubmit hook 
- [x] Remove the input field for the lastUpdated in the docs schema (for the tinacms editor) 

### ✅ Acceptance Criteria 

- [x] Last Edited is automatic 

### 📸 Screenshots 

Cc:

**Comment by isaaclombardssw** [comment]
Got this working but the beforeSubmit hook doesn't seem to add info in to the current visual editing session as far as I can tell (but it's written into the markdown and shows up on refresh). 

I've reproduced this behavior on the bare bones starter at https://github.com/isaaclombardssw/my-tina-app. I've tried debugging via the Form that's passed into the beforeSubmit function (trying to set specific form values) but no dice.

Brady and Gert commented we can probably mark this one as done as it is, and work on fixing the UX (so it shows up immediately) later.

- [ ] Create a PBI for UX improvement after this PBI is marked as completed.
